ext/ArrayLayoutsSparseArraysExt.jl

```diff
@@ -0,0 +1,23 @@
+module ArrayLayoutsSparseArraysExt
+
+using ArrayLayouts
+using ArrayLayouts: _copyto!
+using SparseArrays
+import LinearAlgebra
+
+import Base: copyto!
+
+# ambiguity from sparsematrix.jl
+copyto!(dest::LayoutMatrix, src::SparseArrays.AbstractSparseMatrixCSC) =
```

Comment:
Can you add a test for this line?

Reply:
I've added a test downstream in `BandedMatrices`, so this line is now covered. However, for some reason, the test fails on the CI, although it passes locally for me. Haven't had time to look into it yet

Reply:
Oh, I was looking at the tests for the other PR. The tests do pass for this one, but the coverage isn't being updated for some reason